proposals/gc/MVP.md

```diff
@@ -688,14 +719,17 @@ This extends the [encodings](https://github.com/WebAssembly/function-references/
 | Opcode | Type            | Parameters | Note |
 | ------ | --------------- | ---------- | ---- |
 | -0x10  | `funcref`       |            | shorthand, from reftype proposal |
-| -0x11  | `anyref`        |            | shorthand, from reftype proposal |
+| -0x11  | `externref`     |            | shorthand, from reftype proposal |
+| -0x12  | `anyref`        |            | shorthand |
 | -0x13  | `eqref`         |            | shorthand |
 | -0x14  | `(ref null ht)` | `ht : heaptype (s33)` | from funcref proposal |
 | -0x15  | `(ref ht)`      | `ht : heaptype (s33)` | from funcref proposal |
 | -0x16  | `i31ref`        |            | shorthand |
 | -0x19  | `dataref`       |            | shorthand |
 | -0x1a  | `arrayref`      |            | shorthand |
 | -0x1b  | `nullref`       |            | shorthand |
+| -0x1c  | `nullfuncref`   |            | shorthand |
```

Comment:
Note that `-0x1c` through `-0x1f` are taken by the stringrefs proposal, see https://github.com/WebAssembly/stringref/blob/main/proposals/stringref/Overview.md#binary-encoding.

Reply:
Thanks for pointing out. I opened WebAssembly/stringref#41.

Reply:
FYI, we've implemented these as:
```
| -0x17 | `nullexternref` |
| -0x18 | `nullfuncref`   |
```
to resolve the conflict with the stringref proposal. (Of course this applies to lines 749/750 below as well.)